libsplinter/src/peer/error.rs

```diff
@@ -150,6 +167,7 @@ impl fmt::Display for PeerConnectionIdError {
     }
 }
 
+/// Represents errors raised by trying to update a peer
```

Comment:
Inconsistent - some `struct` comments start with a verb+s; others start with a noun phrase. The Splinter Rustdoc guidelines recommend starting with a noun (for example, "Errors raised by ...").

I didn't flag the other occurences of this inconsistency.

Reply:
hmm okay, I was following the way the registry::errors where described. (most of that rust doc follows the guidelines so i was using it as reference)